MAVProxy/modules/mavproxy_console.py

```diff
@@ -36,6 +36,8 @@ def __init__(self, mpstate):
         self.last_sys_status_errors_announce = 0
         self.user_added = {}
         self.safety_on = False
+        self.unload_check_interval = 0.5 # seconds
```

Comment:
please add to 5s. On some systems the thread creation is very slow. With this 0.5s change it will close the window before it opens

Reply:
done

Reply:
@tridge done

Reply:
@tridge This is an *unload check interval", not the same timeout as we have on some module's load time!

I think I might merge this one anyway - but it really ought to be 0.5s.